src/loaders/ObjectLoader.js

```diff
@@ -610,6 +617,94 @@ THREE.ObjectLoader.prototype = {
 
 			}
 
+			/*
+			 * Now SkinnedMesh could have two bone sets in its children,
+			 * one is made from geometry.bones in SkinnedMesh constroctor
+			 * and another one is made from data.children in this method.
+			 * These two bone sets correspond to the same bones.
+			 * The former would represent a default stand pose while
+			 * the latter would represent a pose updated by skinning.
+			 * To let SkinnedMesh be updated pose, here this logic copies
+			 * the latter one to the former one then removes the latter one.
+			 * If in the latter one there're objects which don't correspond to
+			 * the ones in the former one, they'll be moved to the former one.
+			 */
```

Comment:
@bhouston 
> There is the Skeleton helper for this.

Yes, that one lets you see whether it worked, but not necessarily what went wrong. An online tool (like the Three editor) that lets you load a model from a support request in order to see names & hierarchy might come useful, I figure. 

> We need to get the exporters/workflow fixed... The most important is Blender...

Absolutely. If all others fail can still use Blender as a converter and at least won't have to pay an extra fee on top of the mess. I'm following that topic closely - ready to step in when required. The code quality is very workable - well-structured and not bad at all.